db/indexes_test.go

```diff
@@ -311,3 +326,29 @@ func TestIsIndexerError(t *testing.T) {
 	err = errors.New("err:[5000]  MCResponse status=KEY_ENOENT, opcode=0x89, opaque=0")
 	assert.True(t, isIndexerError(err))
 }
+
+// Drop and reinitialize all indexes
```

Comment:
traditional go style is // dropAndInitializeIndexes drops all sync gateway indexes

the style is to start with the function name first to get godoc to pick up